--- origsrc/kdeadmin-4.4.2/ksystemlog/src/CMakeLists.txt	2009-04-15 05:26:27.000000000 -0500
+++ src/kdeadmin-4.4.2/ksystemlog/src/CMakeLists.txt	2010-04-29 04:24:08.393039400 -0500
@@ -47,8 +47,6 @@ kde4_add_ui_files(ksystemlog_sources
 kde4_add_executable(ksystemlog ${ksystemlog_sources})
 
 target_link_libraries(ksystemlog  
-	${KDE4_KIO_LIBS} 
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_config
 	ksystemlog_open
@@ -64,6 +62,8 @@ target_link_libraries(ksystemlog
 	ksystemlog_postfix 
 	ksystemlog_cups 
 	ksystemlog_samba 
+	${KDE4_KIO_LIBS} 
+	${KDE4_KDEUI_LIBS}
 )
 
 ########### Installation ###############
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/lib/CMakeLists.txt	2010-03-26 12:25:14.000000000 -0500
+++ src/kdeadmin-4.4.2/ksystemlog/src/lib/CMakeLists.txt	2010-04-29 04:24:08.398039700 -0500
@@ -49,7 +49,7 @@ add_dependencies(
 )
 
 target_link_libraries(ksystemlog_lib
-	${KDE4_KDEUI_LIBS}
-        ${KDE4_KIO_LIBS}
 	ksystemlog_config
+	${KDE4_KDEUI_LIBS}
+	${KDE4_KIO_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/acpid/CMakeLists.txt	2008-02-13 09:45:53.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/acpid/CMakeLists.txt	2010-04-29 04:24:08.407040200 -0500
@@ -28,8 +28,8 @@ add_dependencies(
 
 target_link_libraries(
 	ksystemlog_acpid
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_base_mode 
 	ksystemlog_config
+	${KDE4_KDEUI_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/apache/CMakeLists.txt	2008-02-13 09:45:53.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/apache/CMakeLists.txt	2010-04-29 04:24:08.412040500 -0500
@@ -31,8 +31,8 @@ add_dependencies(
 
 target_link_libraries(
 	ksystemlog_apache
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_base_mode
 	ksystemlog_config
+	${KDE4_KDEUI_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/authentication/CMakeLists.txt	2008-02-13 09:45:53.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/authentication/CMakeLists.txt	2010-04-29 04:24:08.416040700 -0500
@@ -26,8 +26,8 @@ add_dependencies(
 
 target_link_libraries(
 	ksystemlog_authentication
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_base_mode 
 	ksystemlog_config
+	${KDE4_KDEUI_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/base/CMakeLists.txt	2008-02-13 09:45:53.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/base/CMakeLists.txt	2010-04-29 04:24:08.420041000 -0500
@@ -30,7 +30,8 @@ add_dependencies(
 
 target_link_libraries(
 	ksystemlog_base_mode
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_config
+	${KDE4_KIO_LIBS}
+	${KDE4_KDEUI_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/cron/CMakeLists.txt	2008-02-13 09:45:53.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/cron/CMakeLists.txt	2010-04-29 04:24:08.424041200 -0500
@@ -28,8 +28,8 @@ add_dependencies(
 
 target_link_libraries(
     ksystemlog_cron
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_base_mode 
 	ksystemlog_config
+	${KDE4_KDEUI_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/cups/CMakeLists.txt	2008-02-13 09:45:53.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/cups/CMakeLists.txt	2010-04-29 04:24:08.429041500 -0500
@@ -43,8 +43,8 @@ add_dependencies(
 
 target_link_libraries(
 	ksystemlog_cups
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_base_mode
 	ksystemlog_config
+	${KDE4_KDEUI_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/daemon/CMakeLists.txt	2008-02-13 09:45:53.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/daemon/CMakeLists.txt	2010-04-29 04:24:08.433041700 -0500
@@ -25,8 +25,8 @@ add_dependencies(
 
 target_link_libraries(
 	ksystemlog_daemon
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_base_mode 
 	ksystemlog_config
+	${KDE4_KDEUI_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/kernel/CMakeLists.txt	2008-02-13 09:45:53.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/kernel/CMakeLists.txt	2010-04-29 04:24:08.437041900 -0500
@@ -23,8 +23,8 @@ add_dependencies(
 
 target_link_libraries(
 	ksystemlog_kernel
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_base_mode
 	ksystemlog_config
+	${KDE4_KDEUI_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/open/CMakeLists.txt	2008-02-13 09:45:53.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/open/CMakeLists.txt	2010-04-29 04:24:08.442042200 -0500
@@ -22,8 +22,8 @@ add_dependencies(
 
 target_link_libraries(
 	ksystemlog_open
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_config
 	ksystemlog_base_mode
+	${KDE4_KDEUI_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/postfix/CMakeLists.txt	2008-02-13 09:45:54.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/postfix/CMakeLists.txt	2010-04-29 04:24:08.446042400 -0500
@@ -26,8 +26,8 @@ add_dependencies(
 
 target_link_libraries(
 	ksystemlog_postfix
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_base_mode
 	ksystemlog_config
+	${KDE4_KDEUI_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/samba/CMakeLists.txt	2008-02-13 09:45:53.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/samba/CMakeLists.txt	2010-04-29 04:24:08.450042700 -0500
@@ -30,8 +30,8 @@ add_dependencies(
 
 target_link_libraries(
 	ksystemlog_samba
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_base_mode
 	ksystemlog_config
+	${KDE4_KDEUI_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/system/CMakeLists.txt	2008-02-13 09:45:53.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/system/CMakeLists.txt	2010-04-29 04:24:08.455043000 -0500
@@ -26,8 +26,8 @@ add_dependencies(
 
 target_link_libraries(
 	ksystemlog_system
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_base_mode
 	ksystemlog_config
+	${KDE4_KDEUI_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/xorg/CMakeLists.txt	2008-02-13 09:45:53.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/xorg/CMakeLists.txt	2010-04-29 04:24:08.459043200 -0500
@@ -28,8 +28,8 @@ add_dependencies(
 
 target_link_libraries(
     ksystemlog_xorg
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_base_mode 
 	ksystemlog_config
+	${KDE4_KDEUI_LIBS}
 )
--- origsrc/kdeadmin-4.4.2/ksystemlog/src/modes/xsession/CMakeLists.txt	2008-02-13 09:45:53.000000000 -0600
+++ src/kdeadmin-4.4.2/ksystemlog/src/modes/xsession/CMakeLists.txt	2010-04-29 04:24:08.463043400 -0500
@@ -29,8 +29,8 @@ add_dependencies(
 
 target_link_libraries(
     ksystemlog_xsession
-	${KDE4_KDEUI_LIBS}
 	ksystemlog_lib
 	ksystemlog_base_mode 
 	ksystemlog_config
+	${KDE4_KDEUI_LIBS}
 )
